core/model/src/driver.rs

```diff
@@ -278,6 +278,7 @@ impl RpcMessage for Enter {
 
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct Exit {
+    sender: String,
```

Comment:
i think you should add sender to the CLI command too now.

You can use `resolve_address()` in `core/payment/src/cli.rs` to remove optional and get your `sender` 

Would call this argument `address` to match the `init` command